src/feature/nodelist/routerstatus_st.h

```diff
@@ -20,6 +21,8 @@ struct routerstatus_t {
   /** Digest of the router's most recent descriptor or microdescriptor.
    * If it's a descriptor, we only use the first DIGEST_LEN bytes. */
   char descriptor_digest[DIGEST256_LEN];
+  ed25519_public_key_t ed25519_signing_key; /**< The ed25519 signing key of
+                                             * this router. */
```

Comment:
This needs to be a pointer: "ed25519_public_key_t *ed25519_signing_key".
Some routers do not have an ed25519 key.

Reply:
0d36d1f514e308d741f2ba232276115458aa86f8